ops/global/main.tf

```diff
@@ -41,12 +47,13 @@ module "okta" {
 
 // App Insights for Azure Functions
 module "insights" {
-  source        = "../services/monitoring"
-  env           = "global"
-  management_rg = data.azurerm_resource_group.rg.name
-  rg_location   = data.azurerm_resource_group.rg.location
-  rg_name       = data.azurerm_resource_group.rg.name
-  tags          = local.management_tags
+  source           = "../services/monitoring"
+  env              = "global"
+  ai_ingest_cap_gb = 100
```

Comment:
What does this flag do? I couldnt find anything for it in google!

Reply:
That's because it's a variable that ends up being used for `daily_data_cap_in_gb` on a `azurerm_application_insights` resource. `100` is the current setting in azure, and trying to terraform on the global config want to change it to 50. This is another small step to getting our global config back in sync.